### PR TITLE
Liked videos

### PR DESCRIPTION
**Pull request by tanveertkd**
Added liked videos to the video library.  
- You can like a video by clicking on the three dots/overflow menu of a video and clicking on the like button
- You must be authenticated to like a video
- Login credentials:  
    - email: adarshbalika@gmail.com
    - password: adarshBalika123

### CODE REVIEW

File: src/components/OverflowMenu/OverflowMenu.jsx [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+                    <li className="overflow-li" onClick={() => likeVideoHandler(video.video)}>
+                        Like
+                    </li>
+                ) : (
+                    <li className="overflow-li" onClick={() => removeFromLikes(video.video)}>
+                        Unlike
+                    </li>
+                )}
+                {/* <li className="overflow-li" onClick={() => likeVideoHandler(video.video)}>
+                    Like
```
Comment (poornima-alugubelly) [inline src/components/OverflowMenu/OverflowMenu.jsx:20]: U can remove unnecessary code

**Review by poornima-alugubelly — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,55 @@
...
+        );
+
+        setButtonText(() => (buttonText === 'Menu' ? 'Close' : 'Menu'));
+
+        setSidebar(() =>
+            sidebar === 'sidebar-container' ? 'sidebar-active' : 'sidebar-container',
+        );
+    };
+
+    const location = useLocation();
```
Comment (horsemaker) [inline src/App.js:27]: If you only need `pathname` you can destructure it right here.

**Review by horsemaker — APPROVED** [review]
The code looks neat! 👌

File: src/components/OverflowMenu/OverflowMenu.jsx (hunk 1 above)
Comment (tanveertkd) [reply src/components/OverflowMenu/OverflowMenu.jsx:20]: Done! Thanks!

File: src/App.js (hunk 2 above)
Comment (tanveertkd) [reply src/App.js:27]: Done. Thanks!

**Pull request merged**